catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -661,9 +668,13 @@ public Response createMetacard(MultipartBody multipartBody, @Context UriInfo req
                     transformer,
                     metacardId);
 
-            Response.ResponseBuilder responseBuilder = Response.ok(content.getInputStream(),
-                    content.getMimeTypeValue());
-            response = responseBuilder.build();
+            try (InputStream inputStream = content.getInputStream()) {
+                Response.ResponseBuilder responseBuilder = Response.ok(inputStream,
+                        content.getMimeTypeValue());
+                response = responseBuilder.build();
+            } catch (IOException e) {
+                LOGGER.debug("Failed to get content in RESTEndpoint");
```

Comment:
What happens if a null response is returned from a REST method? I'm guessing jax-rs will just return a 500 or something, which might not be what we want.